openedx/features/lums/models.py

```diff
@@ -0,0 +1,32 @@
+"""
+Models for LUMS custom Information.
+"""
+from django.contrib.auth.models import User
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+
+class UserProfileExtension(models.Model):
+    """
+    This model is used to store additional custom profile fields.
+    """
+
+    empty = 'punjab'
+    PUNAJB = 'punjab'
+    SINDH = 'sindh'
+    KPK = 'kpk'
+    BALOCHISTAN = 'balochistan'
+
+    PROVINCE_CHOICES = (
+        (PUNAJB, _('Punjab')),
+        (SINDH, _('Sindh')),
+        (KPK, _('Khyber Pakhtunkhwa')),
+        (BALOCHISTAN, _('Balochistan')),
+    )
+
+    user = models.OneToOneField(
+        User, unique=True, db_index=True, related_name='profile_extension', on_delete=models.CASCADE
```

Comment:
nit: we don't need these here `unique=True, db_index=True` django does it by default on `OneToOneField `

Reply:
Done